elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,43 +1,51 @@
 @import "../../pfe-sass/pfe-sass";
 
-$USE-FALLBACK: true;
 $LOCAL: cta;
 
 $pfe-cta__arrow-size: 0.313em;
-$pfe-cta--BackgroundColor--focus: rgba(40,151,240, .2);
+$pfe-cta--BackgroundColor--focus: rgba(40, 151, 240, .2);
```

Comment:
@castastrophe  This color becomes a gotcha on the saturated bands, can we adapt this focus color to be something else when `--theme: saturated`, maybe `rgba(256, 256, 256, 0.3)` ?

![image](https://user-images.githubusercontent.com/456863/69061288-d93cf400-09e6-11ea-8ad3-81bc4d58b631.png)

![image](https://user-images.githubusercontent.com/456863/69061330-e823a680-09e6-11ea-8abb-d88bd9bdbdf4.png)

Also while looking at this, I noticed `--pfe-cta--Color--focus` looks for `--pfe-broadcasted--link--focus`.  Then within the `:host([on="saturated"]:not([pfe-priority])) ` selector, the value of this variable is  set to 
```scss
--pfe-theme--color--link--focus--on-saturated
```
but  in the pfe-colors.css file, the new theme var name is:   
```scss
--theme--link--focus--on-saturated 
```

So my questions are: 
1.  Should the variables be prefixed with pfe?
2.  Should the state (focus) be before the context (on-saturated)?

Reply:
1 - yes I can definitely update the focus state, np.

2 - The code on the pfe-colors page is a demo and not the name of the actual theme variables.  If you look at the sass for that page, I was manually generating them trying to get a preview to work.  The name for the official theme variables is still: `--pfe-theme--color--link--focus--on-saturated`.

Reply:
Updated in the latest commit

Reply:
1. Thx!
2. Ah ok, the different var names might be confusing to others too.
3. Also I'm just realizing... when this branch merges, we will also need to ship an updated major release of the redhat-theme which removes `ui` from the variable names and reverses the order of the context & state names, which will mean we also have to update the var names in the webdrupal repo, correct? 
snapshot from webdrupal:
![image](https://user-images.githubusercontent.com/456863/69113559-52713100-0a51-11ea-8761-d0553ffdbb88.png)

Reply:
2. Yes but I didn't have time to fix the page or get it quite the way I wanted so it's WIP anyway.  We can come back to it another time, it's just a demo page and I added a comment at the top that clarifies that.
3. Yup, the work is already done and tested and an MR open against the theme project.  I'm not sure where in the webdrupal repo we're referencing them but our fallbacks will keep us from seeing any major breaks.